tests/server_event.rs

```diff
@@ -307,7 +306,7 @@ fn event_queue_and_mapping() {
                 ..Default::default()
             }),
         ))
-        .add_server_event::<EntityEvent>(ChannelKind::Ordered)
+        .add_mapped_server_event::<EntityEvent>(ChannelKind::Ordered)
```

Comment:
This should have been a mapped event. Test passed previously because we had the same entity identifiers.
Added an assertion below to prevent this from happening in the future.